test/shell/test_scala_specs2.sh

```diff
@@ -34,6 +34,41 @@ scala_specs2_junit_test_test_filter_everything(){
   done
 }
 
+scala_specs2_junit_test_test_large_unfiltered(){
+  local output=$(bazel test \
+    --nocache_test_results \
+    --test_output=streamed \
+    --test_timeout=10 \
+    test:Specs2LargeTests)
+    
+  local unexpected="TIMEOUT"  
+  if grep "$unexpected" <<<$output; then
+    echo "output:"
+    echo "$output"
+    echo "Not expecting $unexpected in output, but was found."
```

Comment:
WDYT about saying why you're not expecting `unexpected`?
Maybe something like "Expected test to pass but found"?

Reply:
I really don't mind either way, but when someone picks this up, they can probably write a better test anyway, this is sort of a placeholder